packages/lib-classifier/src/store/TutorialStore.js

```diff
@@ -78,6 +78,23 @@ const TutorialStore = types
 
       if (miniCourse) return lastStepSeen === miniCourse.steps.length - 1
 
+      return false
+    },
+
+    get isFirstStep () {
+      if (!!self.active && !!self.activeStep.toString()) {
+        return self.activeStep === 0
+      }
+
+      return false
+    },
+
+    get isLastStep () {
+      if (!!self.active && !!self.activeStep.toString()) {
```

Comment:
Same comments about the conditional here. I don't quite follow what it's checking.